solr/core/src/java/org/apache/solr/schema/ZkIndexSchemaReader.java

```diff
@@ -154,6 +151,26 @@ public void process(WatchedEvent event) {
       }
     }
 
+    @Override
+    public void process(WatchedEvent event) {
```

Comment:
minor: please move this method to immediately before doProcess since it calls it.